ReactWindows/ReactNative.Bridge/IJavaScriptBundleLoader.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Threading.Tasks;
+
+namespace ReactNative.Bridge
+{
+    /// <summary>
+    /// A class that stores JavaScript bundle information and allows the
```

Comment:
going to remove the specific noun reference here, since it just gets in the way of the refactoring tools: "Stores JavaScript bundle information and [...]"